core/src/avm1/object/bitmap_data.rs

```diff
@@ -772,7 +756,8 @@ impl BitmapData {
                         // because of the saturating conversion to u8
                         *noise_c = if c == 3 { 1.0 } else { -1.0 };
 
-                        if (channel_options & (1 << c)) != 0 {
+                        let c = unsafe { ChannelOptions::from_bits_unchecked(1 << c) };
```

Comment:
Could you comment this with a `SAFETY:` comment similar to:
https://doc.rust-lang.org/src/core/slice/iter.rs.html#209
(Trying to get into the habit of doing this for any `unsafe` blocks).